studyzen/FlashCard/FlashCardController.cs

```diff
@@ -0,0 +1,73 @@
+using Microsoft.AspNetCore.Mvc;
+using StudyZen.Common;
+using StudyZen.FlashCards.Requests;
+using Microsoft.AspNetCore.Mvc.ApiExplorer;
+
+
+namespace StudyZen.FlashCards;
+
+[ApiController]
+[Route("[controller]")]
+public sealed class FlashCardsController : ControllerBase
```

Comment:
Consistent naming - lets use `Flashcards`, as it is done in other places

Reply:
naming fixed